web-api/src/ingestion/routes.rs

```diff
@@ -64,6 +72,10 @@ pub(super) fn configure_service(config: &mut ServiceConfig) {
                 .route(web::put().to(put_document_property.error_with_request_id()))
                 .route(web::delete().to(delete_document_property.error_with_request_id())),
         );
+    #[cfg(feature = "ET-3837")]
+    config.service(
+        web::resource("/migration").route(web::patch().to(migrate.error_with_request_id())),
```

Comment:
❓ since this route is not available through the firewall it should only be accesible by us and therefore safe, right? otherwise we have to find another migration trigger.